proxies/live/apiproxy/resources/jsc/WritePathToVariable.js

```diff
@@ -0,0 +1,7 @@
+proxy_pathsuffix    = context.getVariable("proxy.pathsuffix");                 // -> /FHIR/R4/DocumentReference
+target_pathsuffix   = proxy_pathsuffix.replace("/FHIR/R4", "/production");    // -> /production/DocumentReference
+
+// If we don't set this then it will add the "/FHIR/R4/DocumentReference"
+context.setVariable("target.copy.pathsuffix", false);
+// Write the new path into the variable 'target_path'
+context.setVariable("target_path", target_pathsuffix);
```

Comment:
Not sure where we stand on JavaScript conventions in APIgee but should we use camelCase should be used for variables?

Reply:
You're right to push for adherence to standards.  This could be the last time we look at this code for a long time.